### ISSUE TITLE
Feature Request - change size of adjustment controls

### ISSUE THREAD

**Issue by dberg81** [nervetattoo/simple-thermostat#151]
The +/- adjustment controls in compact mode take up a sizable amount of horizontal real estate.  

Would it be possible to allow users to adjust the font size of these controls in a separate variable from target temperature (st-font-size-xl)?

Additionally, would it be possible to change the compact mode +/- adjustment controls to </> characters?  The step_layout column mode uses these same characters displayed vertically and it's a much cleaner look compared to the +/- characters.  
< 1 >   vs.   - 1 +

Using <> characters in compact mode displayed horizontally looks more balanced and wouldn't confuse users which direction increases or deceases the target temperature.

**Comment by nervetattoo** [comment]
Shouldn't be too bad to support custom icon choices for the buttons in both modes. I might look into it the next time I set aside an hour to work on the card, which is not often. Hopefully someone will look into this and submit a PR 😺

**Comment by m4rkireland** [comment]
Yes please.  As mentioned in https://github.com/nervetattoo/simple-thermostat/issues/177, my cards are now overflowing with the `-` being shown underneath the temperature.

Thanks very much.

**Comment by m4rkireland** [comment]
Fixed for now using card-mod:

```
    style: |
      ha-card {
        --st-font-size-xl: 26px;
        --st-font-size-m: 18px;
      }
```

<img width="372" alt="image" src="https://user-images.githubusercontent.com/11231797/102687598-15451880-41e8-11eb-8174-a7d6a44d885a.png">

Thanks

**Comment by shadowdavis** [comment]
> 修复了目前使用卡片模组的问题：
> 
> ```
>     style: |
>       ha-card {
>         --st-font-size-xl: 26px;
>         --st-font-size-m: 18px;
>       }
> ```
> 
> <img alt="图像" width="372" src="https://user-images.githubusercontent.com/11231797/102687598-15451880-41e8-11eb-8174-a7d6a44d885a.png">
> 
> 谢谢

**Comment by shadowdavis** [comment]
> Fixed for now using card-mod:
> 
> ```
>     style: |
>       ha-card {
>         --st-font-size-xl: 26px;
>         --st-font-size-m: 18px;
>       }
> ```
> 
> <img alt="image" width="372" src="https://user-images.githubusercontent.com/11231797/102687598-15451880-41e8-11eb-8174-a7d6a44d885a.png">
> 
> Thanks

`type: custom:simple-thermostat
entity: climate.ac_controller_4eb6
layout:
  mode:
    names: false
    icons: true
    headings: false
  step: column
header: false
control: true
decimals: '0'
step_size: '1'
sensors:
  - entity: sensor.ac_controller_4eb6_humidity
    name: 湿度
  - entity: sensor.xiaomi_fresh_air_ventilator_pm25
    name: PM2.5
  - entity: sensor.xiaomi_fresh_air_ventilator_co2
    name: CO2
style: |
  ha-card {
    --st-font-size-xl: 1px;
    --st-font-size-m: 1px;
  }`


Why can't my configuration work like this. .

**Comment by shadowdavis** [comment]
Why can't my configuration work like this. .
`type: custom:simple-thermostat
entity: climate.ac_controller_4eb6
layout:
  mode:
    names: false
    icons: true
    headings: false
  step: column
header: false
control: true
decimals: '0'
step_size: '1'
sensors:
  - entity: sensor.ac_controller_4eb6_humidity
    name: 湿度
  - entity: sensor.xiaomi_fresh_air_ventilator_pm25
    name: PM2.5
  - entity: sensor.xiaomi_fresh_air_ventilator_co2
    name: CO2
style: |
  ha-card {
    --st-font-size-xl: 1px;
    --st-font-size-m: 1px;
  }`
